client/src/pages/invalid/index.tsx

```diff
@@ -1,12 +1,14 @@
 import React, { ReactElement } from 'react';
 import { Image } from 'semantic-ui-react';
+import TeamModal from '../../components/Modals/EditTeam';
 
 import Error from '../../assets/404-error.png';
 import './styles.scss';
 
 const NotFound = (): ReactElement => (
   <div className="not-found-page">
     <Image size="medium" src={Error} alt="Page not found. Error 404" />
+    <TeamModal />
```

Comment:
There is actually a space for this modal to go now. See the directory designs in figma! there is a little edit button by the column name `Team` in the directory table.